server/src/main/java/com/unosquare/carmigo/entity/Location.java

```diff
@@ -7,16 +7,10 @@
 import javax.persistence.Id;
 import javax.persistence.SequenceGenerator;
 import javax.persistence.Table;
-import lombok.AllArgsConstructor;
-import lombok.Getter;
-import lombok.NoArgsConstructor;
-import lombok.Setter;
+import lombok.Data;
 
 @Entity
-@Getter
-@Setter
-@NoArgsConstructor
-@AllArgsConstructor
```

Comment:
Consider this concerning using @Data when working with @Entity

https://dzone.com/articles/lombok-and-jpa-what-may-go-wrong

Reply:
Removed @Data annotation